Modules/RaidManager/GUI.lua

```diff
@@ -343,11 +347,8 @@ local function CreateRaidDisplay(self)
     end)
     -- end
     -- OnClick handler
-    local OnClickHandler = function(...)
-        local status = self.st.DefaultEvents["OnClick"](...)
-        local args = { ... }
-        local cellFrame = args[2]
-        local button = args[9]
+    local OnClickHandler = function(rowFrame, cellFrame, data, cols, row, realrow, column, table, button, ...)
+        local status = self.st.DefaultEvents["OnClick"](rowFrame, cellFrame, data, cols, row, realrow, column, table, "LeftButton", ...)
```

Comment:
This seems wrong; as I understand it, this will always pass on the click event for any mouse button as a `left` click to the underlying library.

Shouldn't you: 
 - Only pass on the function to the library if you're not handling the event?
 - Always pass on the real mouse button so that lib-st can handle it properly?

Reply:
Library default handler handles selection but it handles it only on left button click. So what we do here is we tell the library that it was a left click so our right click (which shows menu) will also lead to library handling of left click (which is selection). This way you wont need to preselect the entry with left click.